packages/cli/src/executions/executions.controller.ts

```diff
@@ -22,32 +30,66 @@ export class ExecutionsController {
 	}
 
 	@Get('/')
-	async getExecutionsList(req: ExecutionRequest.GetAll) {
+	async getMany(req: ExecutionRequest.GetMany) {
 		const workflowIds = await this.getAccessibleWorkflowIds(req.user);
 
-		return await this.executionService.getExecutionsList(req, workflowIds);
+		if (workflowIds.length === 0) return { count: 0, estimated: false, results: [] };
+
+		return await this.executionService.findMany(req, workflowIds);
+	}
+
+	@Get('/active')
```

Comment:
I'd love to actually get rid of this endpoint, not rename it from `/executions-current` to `/executions/active`.

So this would mean that by default (not filtering for specific stuff), the `/executions` endpoint would return all active + a few finished executions.

I think it's worth coordinating with @alexgrozav the best approach here, and set up a contract that we all can use, since Alex is working on rewriting the execution list.

Reply:
Exactly, this PR is setup for the next story, which is about merging them. In that next story I'll coordinate with Alex.